src/array_api_stubs/_draft/creation_functions.py

```diff
@@ -214,19 +215,36 @@ def eye(
     """
 
 
-def from_dlpack(x: object, /) -> array:
+def from_dlpack(
+    x: object, /, *,
+    device: Optional[device] = None,
+    copy: Optional[bool] = None
+) -> Union[array, Any]:
     """
     Returns a new array containing the data from another (array) object with a ``__dlpack__`` method.
 
     Parameters
     ----------
     x: object
         input (array) object.
+    device: Optional[device]
+        device on which to place the created array. If ``device`` is ``None`` and ``x`` supports DLPack, the output array device must be inferred from ``x.device``. Default: ``None``.
```

Comment:
The consumer will not be able to understand the producer's device object, so `x.device` is a bit misleading here. I suggest something like: "the output array will be on the same device as `x`".

Reply:
I think this is the kind of note that is (array) library developer facing, not end user facing. During the revision of this PR, my feeling is sometimes it is hard to separate them, and sometimes we need to do a better job. Let's apply your suggestion and see what it looks like.

Reply:
Ah okay. I think it's useful to explicitly say "implementors may choose ..." or "implementers need to ...". Or use a `.. note::` if it really needs highlighting. I'm not sure that is done 100% consistently everywhere, but in principle all API docs are for user-observable behavior and should be usable as user-facing docs.

Reply:
LGTM now after commit b855568 - resolving.